Cargo.toml

```diff
@@ -40,6 +40,10 @@ features = ["xlib"]
 version = "2.3.0"
 features = ["xlib"]
 
+[target.armv7-unknown-linux-gnueabihf.dependencies.x11]
```

Comment:
Cool, i'll try that. And maybe I'll update all the other repos where I added the armv7 section...